protocol/messenger_handler.go

```diff
@@ -1928,13 +1939,21 @@ func (m *Messenger) isMessageAllowedFrom(publicKey string, chat *Chat) (bool, er
 	return contact.Added, nil
 }
 
-func (m *Messenger) updateUnviewedCounts(chat *Chat, mentioned bool) {
+func (m *Messenger) increaseUnviewedCounts(chat *Chat, mentioned bool) {
 	chat.UnviewedMessagesCount++
 	if mentioned {
 		chat.UnviewedMentionsCount++
 	}
 }
 
+func (m *Messenger) decreaseUnviewedMessageCounts(chat *Chat, unViewedMessagesCount uint) {
```

Comment:
This is not quite decreasing though, it's setting, it might decrease or it might not change it in case the message was already seen (which is the correct behavior), but the naming of the function is a bit off.